app/views/task/edit.html.erb

```diff
@@ -0,0 +1,16 @@
+<header>
+  <h2>Nothing is so painful to the human mind as a great and sudden change.</h2>
+  <h3> &mdash; Mary Shelley </h3>
+</header>
+
+<!-- Since I couldn't get a restful route for patch to work,
+I ended up going with the custom detailed here. :(  -->
+<%= form_for @task,url:"/tasks/#{@task.id}/edit" do |f| %>
```

Comment:
I'm curious why you didn't include the `form_for` portion of this in the partial as well. Was that due to the different URLs?